docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md

```diff
@@ -1,40 +1,35 @@
 # CREATE TABLE
+ The `CREATE TABLE` statement is used to create a table. The table name must be unique in one database. If the table with the same name in a database is created repeatedly, an error will occur.
```

Comment:
you can just remove the third sentence " If the table with the same name in a database is created repeatedly, an error will occur."